src/main/webapp/portal/main/partials/access-denied.html

```diff
@@ -0,0 +1,13 @@
+<frame-page app-header-title='Access denied' 
```

Comment:
Possibly something to make in the future, maybe make a story for this. The goal of this story is to have a general access denied experience, respecting the `keep it as simple as possible`. Maybe this will come later if the people demand it.